.storybook/global.less

```diff
@@ -0,0 +1,42 @@
+/**
+ * This file contains styles that are used in prod that ultimately
+ * get passed onto Perseus components. This file is imported in the
+ * .storybook/preview.js file so that the components can have the same
+ * styles as prod when viewed within Storybook.
+ */
+
+// Variables
+@offBlack: #21242c;
+@wonderBlocksFontFamily: "Lato", "Noto Sans", "Helvetica", "Corbel", sans-serif;
+
+// Layers
+@layer reset, shared;
```

Comment:
I think this needs to include `legacy`. The rest of our PROD site show:
```
@layer reset, shared, legacy;
```

Reply:
That's true, but I left it out since `@legacy` isn't defined anywhere in perseus styles. Should I add it anyway?

Reply:
Yes, please add it. Although `legacy` is last in the list, and "shouldn't" affect anything when imported due to definition order, we should be consistent with how it is declared in other parts in PROD.

That said, since this is the first time that we have declared and used layers in Perseus, we should verify how this comes across in PROD. Perhaps a ZND would help? I want to make sure that we aren't making sub-layers due to how these styles are imported.